libraries/apollo-ast/build.gradle.kts

```
@@ -4,7 +4,7 @@ plugins {
   antlr
   id("org.jetbrains.kotlin.jvm")
   id("apollo.library")
-  id("com.google.devtools.ksp")
+  kotlin("plugin.serialization")
```

Nit. We're using the `id("")` notation everywhere else

```suggestion
  id("org.jetbrains.kotlin.plugin.serialization")
```